CHANGELOG.rst

```diff
@@ -2,6 +2,41 @@
 CHANGELOG
 =========
 
+2.7.0
+=====
+
+**ENHANCEMENTS**
+
+* FSx Lustre: Exit with error when failing to retrieve FSx mountpoint
+* `sqswatcher`: The daemon is now compatible with VPC Endpoints so that SQS messages can be passed without traversing
+  the public internet.
+
+**CHANGES**
+
+* Upgrade Nice DCV to version 2020.0-8428.
+* Upgrade Intel MPI to version U7.
+* Upgrade NVIDIA driver to Tesla version 440.64.00.
```

Comment:
I'd avoid to call out 'Tesla'

Reply:
It's called out in past upgrades as well, f.i. in 2.5.1. Do you think we should stop doing it?

Reply:
yes